### PR TITLE
Follow-up CODEOWNERS adjustments

### PR DESCRIPTION
**Pull request by parlough**
It seems like later rules from https://github.com/flutter/website/commit/23583e6e068d1e47e91392084eae83f85dd65b21 override the default declared at the top, preventing Shams from approving configuration and root repository files. She needs that access. This PR adjusts the file to return that access.

Also adds Khanh as a code owner for Jekyll plugins and sass files, which aren't often updated, but often need a technical review.

### CODE REVIEW

**Comment by sfshaza2** [comment]
THANKS, @parlough! Yes, that was irritating.

File: CODEOWNERS
Comment (khanhnwin) [inline CODEOWNERS:14]: I believe I _should_ be helping with reviews on tooling as well, @parlough?

File: CODEOWNERS
Comment (khanhnwin) [inline CODEOWNERS:20]: this would include Dockerfile and everything in the root directory? I think I should also be added here?

**Comment by parlough** [comment]
Added @khanhnwin :)

**Comment by parlough** [comment]
Unfortunately CODEOWNERS doesn't have a way to allow for approval without automatically requesting reviews as far as I can tell, even if you specify a team. It's frustrating :/

**Review by khanhnwin — APPROVED** [review]
danke @parlough!

**Comment by khanhnwin** [comment]
@domesticmouse do you have any desire to be included on more PRs? :)

**Review by atsansone — APPROVED** [review]

**Pull request merged**
